source/get-started/primer.txt

```diff
@@ -0,0 +1,157 @@
+   the `Reactive Streams documentation <http://www.reactive-streams.org>`__.
+
+Subscribers
+-----------
+
+The {+driver-short+} API mirrors the :driver:`Java Sync driver
+</java/sync/current/>` API and any methods that cause network 
+I/O return a ``Publisher<T>`` type, where ``T`` is the type of response
+for the operation.
+
+.. note::
+
+   All ``Publisher`` types returned from the API are
+   `cold <https://projectreactor.io/docs/core/release/reference/#reactor.hotCold>`__,
+   meaning that nothing happens until they are subscribed to. So just
+   creating a ``Publisher`` won’t cause any network I/O. It’s not until
+   you call the ``Publisher.subscribe()`` method that the driver executes
+   the operation.
+   
+   Publishers in this implementation are *unicast*. Each
+   ``Subscription`` to a ``Publisher`` relates to a single MongoDB
+   operation, and the ``Publisher`` instance's ``Subscriber`` receives its
+   own specific set of results.
+
+Subscribers in the Quick Start
+------------------------------
+
+In the Quick Start, we have implemented a couple of
+different ``Subscriber`` types. Although this is an
+artificial scenario for reactive streams we do
+block on the results of one example before starting
+the next, so as to ensure the state of the
+database.
+
+1. ``ObservableSubscriber``: The base subscriber class is the
+   `ObservableSubscriber<T> <{+driver-source-gh+}/blob/master/driver-reactive-streams/src/examples/reactivestreams/helpers/SubscriberHelpers.java>`__,
+   a ``Subscriber`` that stores the results of the ``Publisher<T>``. It
+   also contains an ``await()`` method so we can block for results
+   to ensure the state of the database before going
+   on to the next example.
+
+#. ``OperationSubscriber``: An implementation of the
+   ``ObservableSubscriber`` that immediately calls
+   ``Subscription.request()`` when it is subscribed to.
+
+#. ``PrintSubscriber``: An implementation of the ``OperationSubscriber``
+   that prints a message when the ``Subscriber.onComplete()`` method is
+   called.
+
+#. ``ConsumerSubscriber``: An implementation of
+   ``OperationSubscriber`` that takes a ``Consumer`` and calls
+   ``Consumer.accept(result)`` when ``Subscriber.onNext(T result)`` is
+   called.
+
+#. ``PrintToStringSubscriber``: An implementation of
+   ``ConsumerSubscriber`` that prints the string version of the
+   ``result`` when the ``Subscriber.onNext()`` method is called.
+
+#. ``PrintDocumentSubscriber``: An implementation of the
+   ``ConsumerSubscriber`` that prints the JSON version of a ``Document``
+   type when the ``Subscriber.onNext()`` method is called.
+
+Blocking and Non-Blocking Examples
+----------------------------------
+
+As our ``Subscriber`` types contain a latch that is only
+released when the ``onComplete()`` method of the
+``Subscriber`` is called, we can use that latch
+to block on by calling the ``await`` method.
+The following two examples use our auto-requesting
+``PrintDocumentSubscriber``.
```

Comment:
I don't understand this sentence. This is my best guess at a correction:
```suggestion
As our ``Subscriber`` types contain a latch that is only
released when the ``onComplete()`` method of the
``Subscriber`` is called, we can use that latch
to block subscriptions by calling the ``await`` method.
The following two examples use our auto-requesting
``PrintDocumentSubscriber``.
```

Reply:
changed to "further actions"